model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/velocity/velocity_advection.py

```diff
@@ -509,6 +710,12 @@ def run_corrector_step(
 
         self._update_levmask_from_cfl_clipping()
 
+        """
+        z_w_con_c_full (0:nlev-1):
+            Compute the vertical wind with contravariant correction at full levels (cell center) by
+            taking average of from it values at neighboring half levels.
+            z_w_con_c_full[k] = 0.5 (z_w_con_c[k] + z_w_con_c[k+1])
```

Comment:
```suggestion
            z_w_con_c_full[k] = 0.5 (z_w_con_c[k] + z_w_con_c[k+1])
```
I would remove lines like this (e.g. above as well), usually docstrings are used to explain in words what fields are and two cents about what the stencils does